html/semantics/embedded-content/media-elements/track/track-element/cors/support/cors-tester.py

```diff
@@ -1,49 +1,48 @@
 from wptserve.handlers import HTTPException
-import urllib
 
 def main(request, response):
-    if request.method != "GET":
+    if request.method != u"GET":
         raise HTTPException(400, message="Method was not GET")
 
-    if not "id" in request.GET:
+    if not b"id" in request.GET:
         raise HTTPException(400, message="No id")
 
-    id = request.GET['id']
-
-    if "read" in request.GET:
+    id = request.GET[b'id']
+    if b"read" in request.GET:
         data = request.server.stash.take(id)
         if data is None:
             response.set_error(404, "Tried to read data not yet set")
```

Comment:
Alright, after some investigation, this should be **text**.

```suggestion
            response.set_error(404, u"Tried to read data not yet set")
```

Reply:
Sent a PR to clarify the docstring #23555